test/Runtime/POSIX/DirSeek.c

```diff
@@ -43,7 +43,6 @@ int main(int argc, char **argv) {
   // Go to end, then back to 2nd
   while (de)
     de = readdir(d);
-  assert(!errno);
```

Comment:
Instead of removing this `assert`, I would rather set `errno` to `0` before the loop.
I would also put this commit into a separate PR, as we can quickly approve it.